config/intellij-idea-inspections-misc.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why we have misc in name ?

Reply:
This corresponds to name of file that inspections looks for, there should be some similarity here.

Reply:
Is it exact name that idea expects ? Or what do you mean by similarity?

Reply:
```
➜  checkstyle git:(issue-12087) ✗ find config -name "intellij*" 
config/intellij-idea-inspections.xml
config/intellij-idea-inspections-misc.xml
config/intellij-idea-inspection-scope.xml
config/intellij-idea-inspections.properties
➜  checkstyle git:(issue-12087) ✗ ls .idea 
codeStyles    encodings.xml       jarRepositories.xml  scopes          vcs.xml
compiler.xml  inspectionProfiles  misc.xml             uiDesigner.xml  workspace.xml
```
All of the IDEA files that we check in to config directory have `intellij-idea-inspections*` prefix, and this file's original name is `misc.xml`.  So, it makes sense to prefix with common prefix, and keep original file name preserved for  others to understand relevance of file.

Reply:
https://github.com/checkstyle/checkstyle/tree/master/config

Ok, I am not about prefix, I am about suffix "- misc".
Can you name it "-project". Becase it is it is like project file for idea to know nature of it.

Reply:
Or it is suffix that idea requires?
https://github.com/JetBrains/intellij-samples/blob/main/.idea/misc.xml

Reply:
common prefix for all idea config: intellij-idea-inspections
actual file name that idea requires: misc.xml